terragrunt/env/production/api/terragrunt.hcl

```
@@ -0,0 +1,11 @@
+terraform {
+  source = "../../../aws//api"
```

We'll use a different pattern here for specifying the Terraform module source.  Rather than referencing the current branch with a relative path link, we specify the GitHub repo itself and pin it using the version contained in the `infrastructure_manifest`.  That way we know we're getting exactly the Terraform code tagged at that version.
```suggestion
  source = "git::https://github.com/cds-snc/url-shortener//terragrunt/aws/api?ref=${get_env("INFRASTRUCTURE_VERSION")}"
```

---

Done for the production env.